quad_utils/src/trajectory_publisher.cpp

```diff
@@ -4,51 +4,36 @@ TrajectoryPublisher::TrajectoryPublisher(ros::NodeHandle nh) {
   nh_ = nh;
 
   // Load rosparams from parameter server
-  std::string body_plan_topic, foot_plan_continuous_topic, trajectory_topic,
-      trajectory_state_topic, ground_truth_state_topic;
+  std::string body_plan_topic, trajectory_state_topic;
 
   nh.param<std::string>("topics/global_plan", body_plan_topic, "/body_plan");
-  nh.param<std::string>("topics/foot_plan_continuous",
-                        foot_plan_continuous_topic, "/foot_plan_continuous");
-
   nh.param<std::string>("topics/state/trajectory", trajectory_state_topic,
                         "/state/trajectory");
-  nh.param<std::string>("topics/trajectory", trajectory_topic, "/trajectory");
-  nh.param<std::string>("topics/state/ground_truth", ground_truth_state_topic,
-                        "/state/ground_truth");
 
   nh.param<std::string>("map_frame", map_frame_, "map");
   nh.param<std::string>("trajectory_publisher/traj_source", traj_source_,
                         "topic");
   nh.param<double>("trajectory_publisher/update_rate", update_rate_, 30);
-  nh.param<double>("trajectory_publisher/interp_dt", interp_dt_, 0.05);
-  nh.param<double>("trajectory_publisher/playback_speed", playback_speed_, 1.0);
 
   // Setup subs and pubs
   body_plan_sub_ = nh_.subscribe(body_plan_topic, 1,
                                  &TrajectoryPublisher::robotPlanCallback, this);
-  multi_foot_plan_continuous_sub_ = nh_.subscribe(
-      foot_plan_continuous_topic, 1,
-      &TrajectoryPublisher::multiFootPlanContinuousCallback, this);
-  ground_truth_state_sub_ =
-      nh_.subscribe(ground_truth_state_topic, 1,
-                    &TrajectoryPublisher::robotStateCallback, this);
 
   trajectory_state_pub_ =
       nh_.advertise<quad_msgs::RobotState>(trajectory_state_topic, 1);
-  trajectory_pub_ =
-      nh_.advertise<quad_msgs::RobotStateTrajectory>(trajectory_topic, 1);
+
+  // Initialize kinematics object
+  quadKD_ = std::make_shared<quad_utils::QuadKD>();
 }
 
 void TrajectoryPublisher::importTrajectory() {
-  // For Mike
-
   // Clear current trajectory message
   traj_msg_.states.clear();
   traj_msg_.header.frame_id = map_frame_;
   traj_msg_.header.stamp = ros::Time::now();
 
-  // Load the desired balues into traj_msg;
+  // Load the desired values into traj_msg here
+  return;
```

Comment:
I guess we don't need a return statement in a void function? Should we standardize that?